check-links

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Ensures that all links in the repo are valid.
+
+liche --recursive \
+  --concurrency 50 \
+  --timeout 30 \
+  --document-root / \
+  --exclude '(localhost|leader\.mesos|YOUR_SIGNALFX_ORG_ID|YOUR_SIGNALFX_REALM|ingest.*signalfx.com|127\.0\.0\.1|kb\.vmware\.com)' \
```

Comment:
Why is kb.vmware.com excluded?

Reply:
Because it returns HTTP headers that are > 4096 bytes and it errors out since the http client in the checker limits headers to that.  Kind of silly but easier to just exclude.

Reply:
Also support.signalfx.com is excluded because for some reason it doesn't recognize the SSL cert on the domain.